build.mill

```diff
@@ -13,8 +13,12 @@ import mill.scalalib.publish.{Developer, License, PomSettings, VersionControl}
 import org.typelevel.scalacoptions.ScalacOptions.*
 import org.typelevel.scalacoptions.{ScalaVersion, ScalacOptions}
 
-object `mill-scalafix` extends Cross[MillScalafixCross]("1.0.0-RC2", "0.12.0")
```

Comment:
we are dropping support for `0.12.0` which I don't like. But I was unable to cross compile.
Maybe we could use the version specific folders but I think we might as well use a git branch if we need to patch 0.12.x